src/components/MDXContainer.js

```diff
@@ -64,6 +69,7 @@ const components = {
   code: MDXCodeBlock,
   pre: (props) => props.children,
   wrapper: Wrapper,
+  Button,
```

Comment:
I _think_ we may also want to add a shortcode for the `Link` element as well so that it is available for the `as` prop in MDX

Reply:
I had that before, but then I could only get it rendering if I passed it in with an a tag. With the new information I'll make that change :)